app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
                     WHERE p4:i.id = p.id AND mut2:i.id :operator :value
                 )'
             ),
+            'review_status' => new DoctrineSwitchFilterMapping
+            (
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
+                    ),
+                    Presentation::ReviewStatusReceived => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusReceived,
+                        "p.status = 'Received' AND e.published = 0 AND
+                            selp IS NOT NULL AND ( 
+                                (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                                selp.submission_begin_date <= '$now_utc' AND 
+                                selp.submission_end_date >= '$now_utc' AND
+                                selp.selection_begin_date <= '$now_utc' AND 
+                                selp.selection_end_date >= '$now_utc'
+                            )"
+                    ),
+                    Presentation::ReviewStatusInReview => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusInReview,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND
+                            selp IS NOT null AND (
+                            (selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+                            (
+                                (selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
```

Comment:
lets move it to the 3rd place
so if the 2nd place of the case is in review
then we now that we dont need to check the negation of 
NOT ( lockDownPeriod OR (submissionClosed AND selectionOpen) 
bc was tested on 2nd place
and pass ( check how the case function works 
see https://www.w3schools.com/sql/func_mysql_case.asp 
The CASE statement goes through conditions and return a value when the first condition is met (like an IF-THEN-ELSE statement). So, once a condition is true, it will stop reading and return the result.

Reply:
done